### ISSUE TITLE
[Site]: Update info in Events calendar page

### ISSUE THREAD

**Issue by yzeng25** [iresty/iresty.github.io#1767]
### Current State

**Description of the current state/issue**:
Please help update the calendar and monthly meeting invitation link.
**Link to the page**:
https://apisix.apache.org/docs/general/events/
**Screenshots (if relevant)**:

The calendar is empty.
![image](https://github.com/apache/apisix-website/assets/36651058/8e010eac-ced9-400b-98f3-3fcb631fd35a)

This link for monthly meeting invitation is broken.
![image](https://github.com/apache/apisix-website/assets/36651058/556a70d4-a926-474d-a45a-18921254d37f)

Click on this link and you'll get the following:
![image](https://github.com/apache/apisix-website/assets/36651058/f12f5519-eb4c-4a92-ae64-639f8e04d8bb)

### Desired State

**Description of the change you are proposing**:
Calendar and invitation link are updated.

### Would you like to fix this issue?

No. Please assign another contributor

**Comment by SkyeYoung** [comment]
It looks like no community members are hosting community meetings right now, maybe we can consider removing some of the content

**Comment by yzeng25** [comment]
Yeah, do we want to temporarily hide the page or permenantly delete the page and links to the page on other pages?

**Comment by yzeng25** [comment]
Looks like the calendar thing has been disabled recently.

![image](https://github.com/apache/apisix-website/assets/36651058/d4f422fb-8c58-41f9-ae49-07fddb5a49cf)

**Comment by Jitmisra** [comment]
i will fix it https://github.com/apache/apisix-website/pull/1851
